README.md

```diff
@@ -95,6 +97,32 @@ ogcApi:
     tileServer: https://${MY_SERVER}/foo/bar
 ```
 
+### GeoPackage requirements
+
+GoKoala has a few requirements regarding GeoPackages backing an OGC API Features:
+- Each feature table must contain a [RTree](https://www.geopackage.org/guidance/extensions/rtree_spatial_indexes.html) index.
+- Each feature table must contain a BTree spatial index:
+```
+select load_extension('/path/to/mod_spatialite');
```

Comment:
Dit is top :D, te zijner tijd, als we de optimizer hebben aangevuld lijkt het me goed om hier een verwijzing naar de geopackage optimizer op te nemen. En dan de optimizer zo te maken dat deze ook makkelijk op de commandline aan te roepen is (open source etc.).